elasticsearch/run.sh

```diff
@@ -162,13 +162,11 @@ seed_searchguard(){
         -tspass tspass \
         -nhnv \
         -icl ${DEBUG:+-dg}
-    
-    if [ $? -eq 0 ]; then
-      info "Seeded the searchguard ACL index"
-    else
-      error "Error seeding the searchguard ACL index"
-      exit 1
-    fi
+    do
+        warn "Error seeding the searchguard ACL index... retrying in 10 seconds"
+        sleep 10
+    done
+    info "Seeded the searchguard ACL index"
```

Comment:
Is it an infinite loop if the loop ends when `sgadmin` succeeds?

If we don't do it this way, then all we are doing is lengthening the timeout to an arbitrary point that will work for some cases and not for others, right?

The problem today is that it can take a very long to load up all the Elasticsearch indices from slow storage mediums, where ES pods are under-powered, or the load on the node on which they are executing is too high.  Without a successful `sgadmin` init of a new index, the current ES pod will not allow fluentd or kibana access.  Eventually logs will be dropped by fluentd, and Kibana users are given very confusing behavioral messages.

Most of the time we have seen ES after 10 to 20 minutes load all the indices to the point where it exits its `red` state, and then we can manually run `sgadmin` to get things flowing.

Regardless of timeout, the logging system does not work at all if sgadmin does not properly get run on ES pod startup.

If there is another more elegant way to solve this, let's do it.